lib/index.js

```diff
@@ -94,7 +97,7 @@ function makeObject(info) {
         var jsPropertyName = propertyName.replace(/-/g, '_');
 
         Object.defineProperty(constructor.prototype, jsPropertyName, {
-            enumerable: true,
+            configurable: true,
```

Comment:
no, getters and setters are accessors, not data descriptors. They cannot be writable because if you try to write one you trigger the setter ;-)

It would throw otherwise if you have a descriptor that is an accessor but it has writable flag.